colossalai/fx/codegen/activation_checkpoint_codegen.py

```diff
@@ -157,7 +157,7 @@ def _find_offload_regions(nodes: List[Node]):
     current_region = None
 
     for idx, node in enumerate(nodes):
-        if hasattr(node, 'activation_offload') and isinstance(getattr(node, 'activation_offload', False), tuple):
+        if hasattr(node, 'activation_offload') and isinstance(getattr(node, 'activation_offload', False), list):
```

Comment:
`isinstance(getattr(node, 'activation_offload', False), list)` seems weird to me, it seems that `getattr` will return a boolean value, but it is checking isinstance for the type list.

Reply:
Yes the default should be None.
BTW, I change this from tuple to list to simplify the annotation.